### DISCUSSION TITLE
Fantastic work - this is what firefox should be! Thank you so much!!!

### DISCUSSION THREAD

**Discussion by jack3308** [soulhotel/FF-ULTIMA#150] [General]
Hey @soulhotel, this is truly a masterpiece. You've done such a good job with this "theme" (I hesitate to call it that because it's so much more than just a theme). It really makes firefox feel like a modern browser where it hasn't for years now. Seriously good job, cudos on your hard work and thank you for everything you're doing on this. Really really love it.

**Comment by soulhotel** [comment]
Making it feel more modern is the goal, and yet there's still much more I think to be done. I'm glad the theme is useful for you and Thank you very much!
